packages/quantic/force-app/main/default/lwc/quanticResultDate/quanticResultDate.js

```diff
@@ -8,7 +8,7 @@ import {LightningElement, api} from 'lwc';
  * The `QuanticResultDate` component displays a given result date field value.
  * @category Result Template
  * @example
- * <template if:true={result.raw.date}>
+ * <template lwc:if={result.raw.date}>
```

Comment:
Why do we have a `lwc:if` in our example of the usage of the component?

Reply:
not sure why, but I'll remove it, doesn't make much sense here